src/api-client.js

```diff
@@ -44,11 +45,13 @@ export default class ApiClient {
   onRefreshToken: ?Function;
   refreshExpiration: ?number;
   refreshBackend: ?string;
+  isMobile: boolean;
 
   // @see https://github.com/facebook/flow/issues/183#issuecomment-358607052
-  constructor({ server, agent = null, refreshToken, clientId }: ConstructorParams) {
+  constructor({ server, agent = null, refreshToken, clientId, isMobile = false }: ConstructorParams) {
     this.updateParameters({ server, agent, clientId });
     this.refreshToken = refreshToken;
+    this.isMobile = isMobile || false;
```

Comment:
if `isMobile` defaults to false, is the fallback really necessary?

Reply:
necessary for flow ...